spyderlib/plugins/editor.py

```diff
@@ -382,8 +382,10 @@ def __init__(self, parent, ignore_last_opened_files=False):
         
         # Creating template if it doesn't already exist
         if not osp.isfile(self.TEMPLATE_PATH):
-            header = ['# -*- coding: utf-8 -*-', '"""', 'Created on %(date)s',
-                      '', '@author: %(username)s', '"""', '']
+            header = ['#!/usr/bin/env python' + ('2' if PY2 else '3'),

```

Comment:
This line should be here only if we are not on Windows, i.e. `if not os.name == 'nt'`